src/test/java/org/mockitousage/matchers/MatchersTest.java

```diff
@@ -608,7 +608,7 @@ public void same_matcher_and_nulls() {
     public void nullable_matcher() throws Exception {
         // imagine a Stream.of(...).map(c -> mock.oneArg(c))...
         mock.oneArg((Character) null);
-        mock.oneArg(Character.valueOf('€'));
```

Comment:
That statement should work, there's no need to modify the source here. If something is wrong maybe it's the file encoding during the git clone or other editor that changed the encoding. On my clone : 

```
$ file src/test/java/org/mockitousage/matchers/MatchersTest.java
src/test/java/org/mockitousage/matchers/MatchersTest.java: UTF-8 Unicode c program text
```